packages/react-ui/src/stories/widgets/HistogramWidgetUI.stories.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import HistogramWidgetUI from '../../widgets/HistogramWidgetUI';
+
+// This default export determines where your story goes in the story list
+export default {
+  title: 'Widgets/HistogramWidgetUI',
+  component: HistogramWidgetUI,
+};
+
+const DATA = [
+  {
```

Comment:
This is quite generic for our use case.

Something like this will be better:
```javascript
[{
  "tick": "2019-01-01"
  "value": 170155
}]
```